app/partials/footer.html

```diff
@@ -1,3 +1,52 @@
-<div class="footer">
-  <p>♥ from the Yeoman team</p>
-</div>
+<br><br>
+<div class="row" id="other">
+  <div class="col-lg-2 col-md-offset-2">
```

Comment:
This footer list would make more sense if it's in an `<ul>` instead of using `<p>`